### DISCUSSION TITLE
Why did the author encounter this error when using OpenAI's drawing model

### DISCUSSION THREAD

**Discussion by yxl23** [open-webui/open-webui#2251] [Q&A]
![image](https://github.com/open-webui/open-webui/assets/115678682/15fe82f4-d033-426a-b2ba-71c9f853a0dd)

**Comment by silentoplayz** [comment]
https://community.openai.com/t/permissiondeniederror-error-code-403-error-message-you-are-not-allowed-to-sample-from-this-model/654274

**Comment by cheahjs** [comment]
Image generation is not currently very intuitive, https://docs.openwebui.com/tutorial/images#using-image-generation has a tutorial on how to generate an image from the response of a language model.
